src/PostgreSQL.xml

```diff
@@ -18,14 +18,14 @@
       <p>Permission to use, copy, modify, and distribute this software and its documentation for any purpose,
          without fee, and without a written agreement is hereby granted, provided that the above copyright
          notice and this paragraph and the following two paragraphs appear in all copies.</p>
-      <p>IN NO EVENT SHALL THE UNIVERSITY OF CALIFORNIA BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT, SPECIAL,
+      <p>IN NO EVENT SHALL <alt match=".+" name="copyrightHolderLiability">THE UNIVERSITY OF CALIFORNIA</alt> BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT, SPECIAL,
          INCIDENTAL, OR CONSEQUENTIAL DAMAGES, INCLUDING LOST PROFITS, ARISING OUT OF THE USE OF THIS SOFTWARE
-         AND ITS DOCUMENTATION, EVEN IF THE UNIVERSITY OF CALIFORNIA HAS BEEN ADVISED OF THE POSSIBILITY OF
+         AND ITS DOCUMENTATION, EVEN IF <alt match=".+" name="copyrightHolderLiability">THE UNIVERSITY OF CALIFORNIA</alt> HAS BEEN ADVISED OF THE POSSIBILITY OF
          SUCH DAMAGE.</p>
       <p>THE UNIVERSITY OF CALIFORNIA SPECIFICALLY DISCLAIMS ANY WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
```

Comment:
The OSI also uses `$ORGANIZATION` instead of `THE UNIVERSITY OF CALIFORNIA` here.